apps/web/src/services/iam/components/UserManagementRemoveModal/UserManagementOnlyRemoveWorkspaceGroupTypeModal.vue

```diff
@@ -15,17 +15,18 @@ import { ADMIN_ADVANCED_ROUTE } from '@/services/advanced/routes/admin/route-con
 import { userStateFormatter } from '@/services/iam/composables/refined-table-data';
 import { useUserPageStore } from '@/services/iam/store/user-page-store';
 
-
-
 const userPageStore = useUserPageStore();
 const userPageState = userPageStore.state;
-const userPageGetters = userPageStore.getters;
 const userStore = useUserStore();
 
+const selectedUsers = computed(() => userPageState.selectedUsers);
```

Comment:
nitpick: It would be better to just use `userPageState.selectedUsers` directly instead of reassigning the same value!🧐